\section{Origin}

Francois Jacob - Evolution as a tinkerer \cite{jacob_evolution_1977}

"Novelties come from previously unseen association of old material. To create
is to recombine", pp. 3

"Evolution does not produce novelties from scratch. It works on what already
exists, either transforming a system to give it new functions or combining
several systems to produce a more elaborate one." pp. 4

\subsection{Mechanism}

The rise of a functional, novel gene through a de novo route is surely an
unlikely event and requires three steps: transcription, translation, and
functionalization. The order of these steps is by no means obvious.

\subsubsection{Theory}

Two main theories for orphan gene evolution are the continuum and
pre-adaptation theories. Carvunis developed the continuum model
\cite{carvunis_proto-genes_2012}, which describes orphans on a continuum from
random sequence, to proto-gene, to orphan gene, to established gene. The
pre-adaptation model assumes that the barrier to entry is extremely high, such
that only a small subset of genes will become functional. In order to enter the
cycle, they have to have chance properties that prevent them from being toxic.
Specifically, they need have a high intrinsically disordered structure, which
prevents amyloids from forming. This latter model was supported by a study in
mice and yeast \cite{wilson2017young}.

\subsubsection{Transcription}

The first hurdle, at least conceptually, to the rise of a new gene is attaining
transcription.

Techniques for expression analysis, specifically RNA-seq, are extremely
sensitive. There is extensive evidence that transcription is very widespread.

A recent paper from Alba's lab \cite{ruiz2018translation} studied this problem.
Their paper is behind a paywall, and I don't even know what species they were
working with. But they used deep ribosome footprinting and some evolutionary
methods to conclude that translation is common in RNAs thought to be non-coding
and that the majority of the ORFs are under neutral evolution.


There are a few general paths to gaining transcription. 

{\bf Hijack an old gene's transcriptional machinery}. There are a couple ways
an orphan could do this. 1) They could be small ORFs on the mRNA that do not
overlap the primary gene; 2) they could be over-printed, i.e. have an
overlapping ORF in a different frame; 3) they could be ORFs on a functional
non-coding RNA.

{\bf Replace an old gene}. They could completely replace an old gene while
preserving the transcriptional machinery. There are a couple possibilities
here: 1) they could be a frame-shifted old gene; 2) they could evolve from a
pseudogene; 3) they could derive from a deleted CDS. 

{\bf Animated random ORF}. They could be animated by the insertion of a
upstream promoter. The most likely case would be the insertion of an upstream
transposon.  

{\bf Bidirectional transcription}. The de novo theory of origin through
divergent transcription is presented in detail by Wu in \cite{wu2013divergent}.
I haven't carefully read the paper yet ...

\cfig{width=.5\textwidth}{wu2013divergent_fig1}{The majority of mammalian
promoters to not have a TATA-box, and without it they transcribe in either
direction. {\it (A) Transcription factor (TF) binding helps to recruit
TATA-binding protein (TBP) and associated factors, which binds the directional
TATA element in the DNA and orientates RNA Pol II to transcribe downstream DNA.
(B) In the absence of strong TATA elements common of CpG island promoters,
TF-recruited TBP and associated factors binds to low specificity sequences and
forms initiation complexes at similar frequencies in both directions}.
\cite{wu2013divergent}}

{\bf chance promoter}. They could gain a promoter by chance. This specific case
can be recognized through the simplicity of the promoter. A chance promoter
should contain mostly very simple motifs. 


\subsubsection{Translation}

Translation is much harder to assess than transcription. The most
high-throughput method is ribosome footprinting. 


\subsubsection{Functionalization}

Testing for funtionallity of the gene is even harder than testing translation.

How likely is it that a random protein would be beneficial? Or are they highly
toxic? These are vital questions that need to be more thoroughly studied.

Random gene libraries have been used in vivo as a raw source of variation to
evolve novel functions that help E. coli deal with stress
\cite{stepanov2007stress}. This study used 20-mer peptides random libraries and
evolved NiCl2 resistance peptides.

There is evidence that random expression of 65-aa genes in E. coli are actually
quite likely to be beneficial (up to 25\% of clones increase growth rate, and
up to 52\% decrease it) \cite{neme2017random}. This study did not test for
translation of the random sequences, so the effect could be due to just the
transcription. However, they did compare 3 of the beneficial plasmids to the
same plasmid with a early stop insertion: 1 out of 3 showed a change, with the
STOP insertion killing the benefit. This suggests that both the RNA and the
peptide can be beneficial, but is too small of a sample to make a reasonable
estimate of the proportions. 

\cfig{width=\textwidth}{neme2017random-fig1}{{\it Induction of expression
through IPTG drives changes in clone frequency over time.  A) with IPTG, B)
without IPTG. Plots of fold-change (compared to the first cycle) versus mean
counts across pairwise comparisons. Negative fold changes are indicative of
depletion compared to the first cycle, and positive fold changes are indicative
of enrichment compared to the first cycle. Left, center and right panels
indicate comparisons with the 2nd, 3rd and 4th cycle (24 hours per cycle).
Green and red dots indicate clones with significant fold changes (5\% FDR),
positive and negative, respectively. Black crosses indicate clones with
non-significant fold changes. The number in the lower-right corner of each plot
indicates the total number of clones with significant changes (out of 1082).
Both experiments were derived from the same stock culture, and performed
simultaneously. We observe that the induction has a clear effect on the
dynamics of competition and differentiation along the experiment.}
\cite{wu2013divergent}}

One theory for the evolution of functional de novo proteins is that they are
constructed through repeat expansion. It was observed that in mammalian
lineages older proteins have less low-complexity content than younger proteins
\cite{toll2011complexity}.

\subsection{Papers}

\subsubsection{Ruiz-Orera (2015) Origins of de novo genes in human and chimpanzee}

    Citation \cite{ruiz-orera_origins_2015}

    \textbf{Abstract:}

    \begin{verbatim}
    The birth of new genes is an important motor of evolutionary innovation.
    Whereas many new genes arise by gene duplication, others originate at
    genomic regions that do not contain any gene or gene copy. Some of these
    newly expressed genes may acquire coding or non-coding functions and be
    preserved by natural selection. However, it is yet unclear which is the
    prevalence and underlying mechanisms of de novo gene emergence. In order to
    obtain a comprehensive view of this process we have performed in-depth
    sequencing of the transcriptomes of four mammalian species, human,
    chimpanzee, macaque and mouse, and subsequently compared the assembled
    transcripts and the corresponding syntenic genomic regions. This has
    resulted in the identification of over five thousand new transcriptional
    multiexonic events in human and/or chimpanzee that are not observed in the
    rest of species. By comparative genomics we show that the expression of
    these transcripts is associated with the gain of regulatory motifs upstream
    of the transcription start site (TSS) and of U1 snRNP sites downstream of
    the TSS. We also find that the coding potential of the new genes is higher
    than expected by chance, consistent with the presence of protein-coding
    genes in the dataset. Using available human tissue proteomics and ribosome
    profiling data we identify several de novo genes with translation evidence.
    These genes show significant purifying selection signatures, indicating
    that they are probably functional. Taken together, the data supports a
    model in which frequently-occurring new transcriptional events in the
    genome provide the raw material for the evolution of new proteins.
    \end{verbatim}

\subsubsection{Chen (2015) Emergence, retention and
selection: a trilogy of origination for functional de novo proteins from
ancestral LncRNAs in primates}

    Citation \cite{chen_emergence_2015}

    Suggests that the evolution of de novo genes from lncRNAs is not beyond
    neutral expectation. Yet many of these genes in humans are under positive
    selection, implying they have found functional roles.

    They found the lncRNAs with de novo orphans had much higher GC contents
    than other lncRNAs or other coding genes. One possible explanation is that
    the GC bias reduces the risk of stop codon random appearance. Thus high GC
    content ORFs have longer lifetimes. They have fewer fragile codons --
    codons convertible to a stop codon by a single missense mutation.

    Counted a missing ORF as one where at least one frame disruption led to a
    loss of at least 30\% of the original length.

\subsubsection{Reinhardt (2013) De Novo ORFs in Drosophila Are Important to
Organismal Fitness and Evolved Rapidly from Previously Non-coding Sequences}
    
    Citation \cite{reinhardt_novo_2013}

    previously known \cite{levine_novel_2006, zhou_origin_2008} de novo D.
    melanogaster orphans. Two (CG32582, CG32690) were ncRNAs established 5
    million years before gaining an ORF and coding functionality.
    Trascription and translation may have co-occurred in the other four
    (CG33235, CG31406, CG31909, CG34434).

\subsubsection{Ding (2012) Origins of New Genes and Evolution of Their
Novel Functions}
    
    Citation \cite{ding_origins_2012}

\subsubsection{Hotopp (2007) Widespread Lateral Gene Transfer from
Intracellular Bacteria to Multicellular Eukaryotes}

    Citation \cite{hotopp_widespread_2007}

Fly gay gene: It does not encode proteins, and, surprisingly, abolishment
of splicing sites at its recruited intron in the male-specific transcript
leads to male-male courtship behavior. This indicates a novel role for
sphinx in enhancing the heterosexual courtship of D. melanogaster (Dai et
al. 2008)

\subsection{Organisms}

\subsubsection{Silkworm}

  \cite{sun_identification_2015}

  738 orphan genes were identified, 31\% appear to have been derived from
  transposable elements, 5\% from gene duplication followed by rapid
  divergence, 5 genes predicted to have arisen from non-coding regions (de
  novo). The 5 de novo genes do not appear to be essential (RNAi results).

\subsubsection{Drosophila}

    \cite{levine_novel_2006, begun_evidence_2006}; a review of Dm de novo
    genes \cite{zhou_origin_2008}

\subsubsection{Yeast}

    one de novo gene \cite{cai_novo_2008}; de novo on old genes antisense
    strand \cite{li_novo_2010}

\subsubsection{Humans} 

    A brain gene with a thorough description of its means of de novo origin
    \cite{li_human-specific_2010}. 3 de novo genes (with protein product
    support) found in humans despite extremely stringent pipeline
    \cite{knowles_recent_2009}.

\subsubsection{Mouse}

    Testes specific, sperm motility enhancing, 3 exon, \textit{mus
    musculus}-specific gene appearing in inter-genic region (not predicted
    to be translated, but gives evidence of rise of functional, transcribed
    and spliced gene from cryptic signals) \cite{heinen_emergence_2009}. 69
    mouse and 6 rat well-supported de novo genes \cite{murphy_novo_2012}.

\subsubsection{Plasmodium}

    Reports 13 de novo genes (arising in well-established non-genic space)
    five of which have introns (authors suggest intronization occurs before
    transaltability) \cite{yang_novo_2011}

\subsection{Means}

\subsubsection{Rearangement}

A review by Light et al. introduces the idea that many orphans may arise in the
subtelomeric regions where evolutionary rates are high and rearangements
frequent \cite{light_orphans_2014}.

  \cfig{width=0.8\textwidth}{light-orphan-counts}{%
    Number of orphans in various species based on protein blast
    \cite{light_orphans_2014}
  }

\subsubsection{Cryptic signals} 

    Many de novo genes can be traced back to non-genic, non-transcribed
    sequence. The appearance of splicing signals and cryptic promoters can
    be traced in multiple alignments \cite{heinen_emergence_2009,
    yang_novo_2011, knowles_recent_2009}. The gene may be functional as a
    ncRNA \cite{heinen_emergence_2009}.

\subsubsection{Horizontal Gene Transfer (HGT)}

    Widespread in angiosperm mitochondria. This was first noted in 2003 by
    Bergthorsson et al \cite{bergthorsson_widespread_2003}. A massive case was
    reported in 2013 where 6 genome equivalents was added to the Amborella
    mitochondrial genome from green algae, mosses and other angiosperms
    \cite{rice_horizontal_2013}.

    In arthropods, the genomic material, and sometimes even whole genomes, can
    be acqured from the intracellular parasite Wolbachia pipientis
    \cite{hotopp_widespread_2007}.

\subsubsection{lncRNA piggy-backing}

    Many long non-coding RNAs (lncRNAs) are functional. There long lifespans
    and stable expression could act as a foundation for coding orphan gene
    evolution. According to Neme \cite{neme_phylogenetic_2013} many orphan are
    translated from the antisense regions of older genes. Long non-coding
    natural antisense transcripts (lncNATs) are extremely common (~70\% of
    Arabidopsis mRNAs have one \cite{wang_genome-wide_2014}), making them a
    logical transitional state between untranscribed sequence and coding gene.
    There potential for long-term expression and their frequent regulatory
    coupling with their "host" gene may catalyze the transition.

\subsubsection{Overprinting}

    See notes on lncNATs in prior section. Overprinting is a major topic of
    Neme's paper \cite{neme_phylogenetic_2013}.
